build.gradle

```diff
@@ -176,6 +176,7 @@ dependencyLicenses.enabled = false
 loggerUsageCheck.enabled = false
 testingConventions.enabled = false
 thirdPartyAudit.enabled = false
+publishNebulaPublicationToMavenLocal.enabled = false
```

Comment:
why this task has been disabled?

Reply:
After changing the group to `org.opensearch` under allprojects, the publishToMavenLocal failed, the reason is this task found the two groupId not same: one is `org.opensearch` under allprojects, another is `org.opensearch.plugin` under publishing like below:
<img width="1157" alt="Screenshot 2024-01-31 at 08 29 05" src="https://github.com/opensearch-project/skills/assets/114061051/3003ca58-c528-4925-a144-7744d686caff">
But we need to change group to `org.opensearch` under allprojects, to avoid publishing jar files to maven local based on EE team's ask. So we need to disable this task to by pass the error.